identity-iota/src/tangle/message/message_version.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use num_traits::FromPrimitive;
+
+use crate::Error;
+
+/// Indicates the version of a DID Message.
+#[derive(Copy, Clone, Debug, Ord, PartialOrd, Eq, PartialEq, num_derive::FromPrimitive)]
+pub enum DIDMessageVersion {
+  V1 = 1,
+}
+
+pub(crate) const CURRENT_MESSAGE_VERSION: DIDMessageVersion = DIDMessageVersion::V1;
```

Comment:
Consider making this an associated constant of the `DIDMessageVersion` struct like this:
```rust
impl DIDMessageVersion {
    const CURRENT: Self = DIDMessageVersion::V1 
}
```